client/Makefile

```diff
@@ -32,9 +29,9 @@ installer: build
 
 setup-venv:
 	@if [ ! -d venv ]; then virtualenv venv; fi
-	venv/bin/pip install -q flake8==2.4.0
 
 test-style: setup-venv
+	venv/bin/pip install -q flake8==2.4.0
```

Comment:
:shrug: `setup-venv` creates the python virtualenv, and `test-style` is the only Makefile target that uses `flake8`, so it seemed more appropriate to localize it, but I can revert.